tools/coreml/converter/_layers.py

```diff
@@ -361,7 +395,7 @@ def convert_convolution(net, node, module, builder):
         stride_height=stride_height,
         stride_width=stride_width,
         border_mode=border_mode,
-        groups=1,
+        groups=n_groups,
```

Comment:
Is it possible to add a unit test case for this?

Reply:
Added in c94bbb8